### DISCUSSION TITLE
rsp1b support

### DISCUSSION THREAD

**Discussion by vq2bc** [AlexandreRouma/SDRPlusPlus#1452] [General]
In windows Sdrplay rsp1a works ok but rspib loads program and runs but complains about "No device available" Any fix on way?

**Comment by AlexandreRouma** [comment]
Device is not supported

**Reply by vq2bc** [reply]
Will a future release support it?

**Reply by AlexandreRouma** [reply]
Can't support hardware I don't have, and can't afford to buy hardware I won't need for anything other than implementing support.

**Reply by vq2bc** [reply]
Thank you for replies and understood...
